src/main/java/org/mvc/models/IncomingMessage.java

```diff
@@ -0,0 +1,23 @@
+package org.mvc.models;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class IncomingMessage {
+    private String user;
+    private String message;
+
+    public IncomingMessage(
+            @JsonProperty("user") String user,
```

Comment:
What's appoint of using jsonProperty if objectmapper by default names properties after the attribute's name?

Reply:
Redundancy removed